filter.py

```diff
@@ -1,51 +1,54 @@
-#--------Helper Functions to be used for sorting and filtering-------#
+"""Helper Functions to be used for sorting and filtering."""
 import datetime
 
 
-# Filters by all of the criteria shown below, returns an array of filtered releases
-def filter(data, state, label, start_date, end_date, datetype):
+def filter_releases(data, state, label, start_date, end_date, datetype):
```

Comment:
Please specify what each argument should be and what kind of object type is expected. You can use the following format as an example
https://github.com/kubernetes/test-infra/blob/master/boskos/janitor/janitor.py#L87

Reply:
data should be renamed releases ? you might want to filter by labels and states.

Reply:
Also please define what is the return format for each method.